README.md

```diff
@@ -224,6 +224,17 @@ Once `analytics.yml` and friends are populated, the associated models will be
 automatically instrumented to send updates to BigQuery via ActiveRecord
 callbacks.
 
+#### Transactions
+
+In order to correctly record data changes in transactions, specifically in regard to rollbacks and updates, we include the TransactionChanges module in all ActiveRecord models.
+
+This can conflict with other gems who modify the `active_record#write_attribute` method, e.g. `globalize` gem. The solution is to require this gem first, e.g.
+
+```
+gem 'dfe-analytics'
+gem 'globalize' require: ['dfe-analytics', 'globalize']
```

Comment:
Do we know if this is an existing problem on any project (Which we probably don't!)? If it is then what will be implications if we don't fix it ? My feeling is that the issue we're trying to fix is not fixed and goes unnoticed.

Reply:
I don't have much insight on this, globalize is not the only gem which may define write_attribute. I would hope we would see some failing tests in the build if there was a conflict?